### PR TITLE
Mimic identity changes for ele

### PR DESCRIPTION
**Pull request by lekhajee**
This a temporary change that implements only a few rbac like roles (required by ele tests). This should  exist only until RBAC is globally applied for users within mimic.

Eventually tests within ele are going to be changed to hit the mimic identity's control plane to specify the role/behavior expected for a tenant.

### CODE REVIEW

File: mimic/canned_responses/mimic_presets.py [hunk 1]
```diff
@@ -19,5 +19,19 @@
...
+                                              " in seconds"},
+               "identity": {
+                   "maas_admin_roles": [
+                       "this_is_an_impersonator_token",
+                       "this_is_an_impersonator_token_also",
+                       "impersonate_watson",
+                       "impersonate_creator",
+                       "this_is_an_impersonator_token_also_2",
+                       "impersonate_foo_token"],
+               "racker_token": ["this_is_a_racker_token"],
```
Comment (cyli) [inline mimic/canned_responses/mimic_presets.py]: Non-blocking - I think the indentation starting from here might be kind of messed up (should it be lined up with `"maas_admin_roles"`?)

File: mimic/rest/auth_api.py [hunk 2]
```diff
@@ -190,6 +191,46 @@ def validate_token(self, request, token_id):
             response["access"]["RAX-AUTH:impersonator"] = impersonator_user_role(
```
Comment (cyli) [inline mimic/rest/auth_api.py]: Non-blocking - just wanted to make sure that it's ok for this logic that sets the impersonation token to get run even if it's one of the tenants in the hard-coded maas roles?

**Comment by cyli** [comment]
Just some nitpicks - particularly the indentation.  But other than that, LGTM if this is needed right away.  :)

**Comment by glyph** [comment]
I am guessing this is RFR since @cyli is already reviewing it :).

File: mimic/test/test_auth.py [hunk 3]
```diff
@@ -915,6 +916,114 @@ def test_response_for_validate_impersonated_token_multiple_users(self):
         self.assertEqual(json_body6["access"]["RAX-AUTH:impersonator"]["name"],
                          json_body1["access"]["user"]["name"])
 
+    def test_response_for_validate_token_with_maas_admin_role(self):
+        """
+        Test to verify :func: `validate_token` when the token_id provided
+        is of an maas admin user specified in `mimic_presets`.
+        """
+        core = MimicCore(Clock(), [ExampleAPI()])
+        root = MimicRoot(core).app.resource()
```
Comment (glyph) [inline mimic/test/test_auth.py]: Given that each of these tests starts with *exactly* these two lines, maybe you could factor out just a simple function so we could do `core, root = core_and_example_root()`?

File: mimic/rest/auth_api.py (hunk 2 above)
Comment (lekhajee) [inline mimic/rest/auth_api.py]: Yes, we the token being validated was got from impersonation, we expect the impersonator role to exist.

**Comment by cyli** [comment]
LGTM!

**Comment by lekhajee** [comment]
@cyli: Thanks much for the quick review and merge :)

**Pull request merged**